### ISSUE TITLE
Error when running example

### ISSUE THREAD

**Issue by gavinmdouglas** [maxibor/sourcepredict#2]
Hi @maxibor,

I'm reviewing your tool for JOSS (see: https://github.com/openjournals/joss-reviews/issues/1540) and will be making at least one other issue with my full review soon.

I just ran into a problem when running the example command though so I thought I should mention this immediately:
`sourcepredict -t 6 dog_test_sample.csv`

`
  File "/home/gavin/local/miniconda3/envs/sourcepredict/bin/sourcepredict", line 96, in <module>
    a.add_unknown(alpha=ALPHA)
  File "/home/gavin/local/miniconda3/envs/sourcepredict/bin/sourcepredictlib/forest.py", line 37, in add_unknown
    self.unknown.columns = ['UNKNOWN']
  File "/home/gavin/local/miniconda3/envs/sourcepredict/lib/python3.6/site-packages/pandas/core/generic.py", line 5080, in __setattr__
    return object.__setattr__(self, name, value)
  File "pandas/_libs/properties.pyx", line 69, in pandas._libs.properties.AxisProperty.__set__
  File "/home/gavin/local/miniconda3/envs/sourcepredict/lib/python3.6/site-packages/pandas/core/generic.py", line 638, in _set_axis
    self._data.set_axis(axis, labels)
  File "/home/gavin/local/miniconda3/envs/sourcepredict/lib/python3.6/site-packages/pandas/core/internals/managers.py", line 155, in set_axis
    'values have {new} elements'.format(old=old_len, new=new_len))
ValueError: Length mismatch: Expected axis has 2 elements, new values have 1 elements
`

This is the output of `conda list`, which I see lists v0.1.1 rather than v0.3.3 so perhaps the installation instructions should be changed so that the latest version is installed?

```
packages in environment at /home/gavin/local/miniconda3/envs/sourcepredict: 
Name                    Version                   Build  Channel  
_libgcc_mutex             0.1                        main 
blas                      1.0                         mkl 
ca-certificates           2019.5.15                     0
certifi                   2019.6.16                py36_0
intel-openmp              2019.4                      243
joblib                    0.13.2                   py36_0
libedit                   3.1.20181209         hc058e9b_0
libffi                    3.2.1                hd88cf55_4
libgcc-ng                 9.1.0                hdf63c60_0
libgfortran-ng            7.3.0                hdf63c60_0
libstdcxx-ng              9.1.0                hdf63c60_0
mkl                       2019.4                      243
mkl-service               2.0.2            py36h7b6447c_0
mkl_fft                   1.0.12           py36ha843d7b_0
mkl_random                1.0.2            py36hd81dba3_0
ncurses                   6.1                  he6710b0_1
numpy                     1.16.4           py36h7e9f1db_0
numpy-base                1.16.4           py36hde5b4d6_0
openssl                   1.1.1c               h7b6447c_1
pandas                    0.24.2           py36he6710b0_0
pip                       19.1.1                   py36_0
python                    3.6.8                h0371630_0 
python-dateutil           2.8.0                    py36_0 
pytz                      2019.1                     py_0 
readline                  7.0                  h7b6447c_5 
scikit-learn              0.21.2           py36hd81dba3_0 
scipy                     1.2.1            py36h7c811a0_0 
setuptools                41.0.1                   py36_0 
six                       1.12.0                   py36_0 
sourcepredict             0.1.1                         0    maxibor 
sqlite                    3.28.0               h7b6447c_0 
tk                        8.6.8                hbc83047_0 
wheel                     0.33.4                   py36_0 
xz                        5.2.4                h14c3975_4 
zlib                      1.2.11               h7b6447c_3 
```

**Comment by maxibor** [comment]
Hi @gavinmdouglas,
First of all, thanks for taking the time to review SourcePredict for JOSS :)

There was indeed a conda channel missing in the installation intructions, it should be fixed by now with 0f75504.
The new install instructions are `conda install -c conda-forge -c etetoolkit -c bioconda -c maxibor sourcepredict`

**Comment by gavinmdouglas** [comment]
v0.33 is now installed correctly and the example command works.
